sdks/python/apache_beam/io/avroio.py

```diff
@@ -226,7 +226,7 @@ def read_meta_data_from_file(f):
     meta = header['meta']
 
     if datafile.CODEC_KEY in meta:
-      codec = meta[datafile.CODEC_KEY]
+      codec = meta[datafile.CODEC_KEY].decode('utf-8')
```

Comment:
When running the avroio tests in Python 3, this results in the following error for me:
```
File "/home/robbe/workspace/beam/sdks/python/apache_beam/io/avroio.py", line 355, in _decompress_bytes
    raise ValueError('Unknown codec: %r' % codec)
ValueError: Unknown codec: 'null'
```
I've used another approach and have fixed all but 2 Python 3 errors in avroio. see #7644.

Reply:
Approach in https://github.com/apache/beam/pull/7644 makes sense to me. Once merged - @charlesccychen  PTAL if you still encounter the errors and rebase the PR on top of master. Thank you.